### ISSUE TITLE
MAS 1.4.13 - Content on Hover or Focus

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#7531]
Content that is revealed on hover or focus (tooltips, dropdowns, etc) must be
- dismissable: the content can be dismissed or does not hide/interfere with other content
- hoverable: if the content displays on hover of another element, it must be able to be hovered over
- persistent: the content must not disappear until the hover or focus is removed

Fortunately, most of our controls are automatically handled by the browser and should be compliant, but some things, such as the Bootstrap tooltips and the custom multiple version dropdown for deprecation need to be analyzed.

**Comment by scottbommarito** [comment]
I was unable to find any violations on our site.
